components/filter/filter.js

```diff
@@ -356,16 +357,8 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		}
 	}
 
-	_formatFilterCount(count, max) {
-		if (count === 0) return;
-
-		let number = count;
-		if (count === max) {
-			number = 'all';
-		} else if (count >= 100) {
-			number = 'max';
-		}
-		return this.localize('components.filter.filterCount', { number: number });
+	_formatFilterCount(count) {
```

Comment:
This method seems kind of pointless now -- could it just be removed?

Reply:
I think you need to get the latest version of this file, this function changed recently.  And then for now can we leave this function as is and just use this above (no need for the `span`):
````
<d2l-count-badge number="${dimension.appliedCount}"></d2l-count-badge>
```
This will leave the filter button working for now, and once I switch that out we can delete this function